src/routes/signup.tsx

```diff
@@ -90,16 +103,25 @@ const SignUpForm = () => {
             minLength={minPasswordLength}
             maxLength={maxPasswordLength}
           />
-          <Dropdown<Realm>
-            value={realms.find((r) => r.id === realmId)}
-            emptyLabel="Select a realm"
-            class={dropdownClass}
-            options={realms}
-            required
-            onChange={(v) => setRealmId(v.id)}
-            getKey={(v) => v.id}
-            getText={(v) => v.name}
-          />
+          <div class={realmElementsStyle}>
+            <Dropdown<Realm>
+              value={realms.find((r) => r.id === realmId)}
+              emptyLabel="Select a realm"
+              class={dropdownClass}
+              options={realms}
+              required
+              onChange={(v) => setRealmId(v.id)}
+              getKey={(v) => v.id}
+              getText={(v) => v.name}
+            />
+            <div class={textStyle}>
+              Don&apos;t see your team? Ask a captain or mentor to make a new
+              realm.{' '}
+            </div>
+            <Button disabled={isLoading} href={'/newrealm'}>
+              New Realm
+            </Button>
```

Comment:
To me, this "new realm" button comes across as way too visually prominent. I could totally see someone filling out this signup form and clicking the big "new realm" button accidentally instead of clicking "sign up" as intended.

Also, I think we can un-bold the "don't see your team..." text.

I think making the new realm be a link in the text would reduce its visual prominence, making it more clear that "sign up" is the primary action for this page. Here's what that would look like:

![image](https://github.com/Pigmice2733/peregrine-frontend/assets/13206945/02ce4031-90d5-405b-902d-87962ece00cb)